<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieTracker</class>
 <widget class="QDialog" name="MovieTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie tracker</string>
  </property>
  <widget class="QLabel" name="App_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>1311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color : grey;
	color : white;
}</string>
   </property>
   <property name="text">
    <string>Movie Tracker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>0</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background : transparent;
	color : white;
}</string>
   </property>
   <property name="text">
    <string>&lt;- Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="App_name_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>680</y>
     <width>1321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color : grey;
	color : white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color : rgb(228, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>690</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color : rgb(175, 244, 198);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1311</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView
{
	background-color : white;
	color : black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="info_button">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background : transparent;
	color : white;
}</string>
   </property>
   <property name="text">
    <string>Detailed info</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="numberOfMoviesText">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>15</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	background-color : transparent;
	color : white;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color : rgb(228, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color : rgb(228, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>700</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color : rgb(228, 204, 255);
}</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>700</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	background-color : rgb(188, 227, 255);
	color : black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
